morphman/common/centerline_operations.py

```diff
@@ -862,3 +862,32 @@ def filter_centerlines(centerlines, diverging_centerline_end):
     filtered_centerlines = vtk_merge_polydata(remaining_centerlines)
 
     return filtered_centerlines
+
+
+def reverse_centerline(centerline_to_reverse):
```

Comment:
Would it not be better to use WrapDataObject for this? These three lines could replace this function.
```
from vtk.numpy_interface import dataset_adapter as dsa
centerline = dsa.WrapDataObject(centerline_to_reverse)
centerline.Points = centerline.Points[::-1]

return centerline.VTKObject
```

Reply:
Note, I did not test this, but I'm quite sure that it should work.

Reply:
Tested this and it does definitely work! Nice improvement ;)